internal/config/profile.go

```diff
@@ -101,7 +102,7 @@ func List() []string {
 
 // Exists returns true if there are any set settings for the profile name.
 func Exists(name string) bool {
-	return len(viper.GetStringMap(name)) > 0
+	return search.StringInSlice(List(), name)
```

Comment:
the original method was giving me problems due to the same problem we have of viper over writing configs